packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/KeyValuesTable.tsx

```diff
@@ -153,12 +153,14 @@ export default function KeyValuesTable(props: KeyValuesTableProps) {
                     className="KeyValueTable--copyIcon"
                     copyText={row.value}
                     tooltipTitle="Copy value"
+                    buttonText="Copy"
                   />
                   <CopyIcon
-                    className="KeyValueTable--copyIcon"
+                    className="KeyValueTable--copyIcon copyIcon"
```

Comment:
`copyIcon` - why is this needed? It's not even defined in the css file.

Reply:
Sorry my bad I thought to use it in css of keyValueTable.css but later realised to use the css of Copyicon.css. Finally it is of no use.Let me make the changes.